test/config.bats

```diff
@@ -720,82 +731,10 @@ function teardown() {
 	bundle-verify "$BUNDLE"
 
-	sane_run jq -SMr '.annotations["com.cyphar.test"]' "$BUNDLE/config.json"
+	sane_run jq -SMr '.annotations["org.opencontainers.image.exposedPorts"]' "$BUNDLE/config.json"
 	[ "$status" -eq 0 ]
-	[[ "$output" == "1" ]]
-
-	sane_run jq -SMr '.annotations["com.cyphar.empty"]' "$BUNDLE/config.json"
-	[ "$status" -eq 0 ]
-	[[ "$output" == "" ]]
-
-	image-verify "${IMAGE}"
-}
-
-@test "umoci config --config.label --manifest.annotation" {
-	BUNDLE="$(setup_tmpdir)"
-
-	# Modify none of the configuration.
-	umoci config --image "${IMAGE}:${TAG}" --tag "${TAG}-new" \
-		--clear=config.labels --clear=manifest.annotations \
-		--config.label="com.cyphar.label_test={another value}" --config.label="com.cyphar.label_empty=" \
-		--manifest.annotation="com.cyphar.manifest_test= another valu=e  " --manifest.annotation="com.cyphar.manifest_empty="
-	[ "$status" -eq 0 ]
-	image-verify "${IMAGE}"
-
-	# Unpack the image again.
-	umoci unpack --image "${IMAGE}:${TAG}-new" "$BUNDLE"
-	[ "$status" -eq 0 ]
-	bundle-verify "$BUNDLE"
-
-	sane_run jq -SMr '.annotations["com.cyphar.label_test"]' "$BUNDLE/config.json"
-	[ "$status" -eq 0 ]
-	[[ "$output" == "{another value}" ]]
-
-	sane_run jq -SMr '.annotations["com.cyphar.label_empty"]' "$BUNDLE/config.json"
-	[ "$status" -eq 0 ]
-	[[ "$output" == "" ]]
-
-	sane_run jq -SMr '.annotations["com.cyphar.manifest_test"]' "$BUNDLE/config.json"
-	[ "$status" -eq 0 ]
-	[[ "$output" == " another valu=e  " ]]
-
-	sane_run jq -SMr '.annotations["com.cyphar.manifest_empty"]' "$BUNDLE/config.json"
-	[ "$status" -eq 0 ]
-	[[ "$output" == "" ]]
-
-	image-verify "${IMAGE}"
-}
-
-# XXX: This is currently not in any spec. So we'll just test our own behaviour
-#      here and we can fix it after opencontainers/image-spec#479 is fixed.
-@test "umoci config --config.label --manifest.annotation [clobber]" {
-	BUNDLE="$(setup_tmpdir)"
-
-	# Modify none of the configuration.
-	umoci config --image "${IMAGE}:${TAG}" --tag "${TAG}-new" \
-		--clear=config.labels --clear=manifest.annotations \
-		--config.label="com.cyphar.test= this_is SOEM VALUE" --config.label="com.cyphar.label_empty=" \
-		--manifest.annotation="com.cyphar.test== __ --a completely different VALuE    " --manifest.annotation="com.cyphar.manifest_empty="
-	[ "$status" -eq 0 ]
-	image-verify "${IMAGE}"
-
-	# Unpack the image again.
-	umoci unpack --image "${IMAGE}:${TAG}-new" "$BUNDLE"
-	[ "$status" -eq 0 ]
-	bundle-verify "$BUNDLE"
-
-	# Manifest beats config.
-	sane_run jq -SMr '.annotations["com.cyphar.test"]' "$BUNDLE/config.json"
-	[ "$status" -eq 0 ]
-	[[ "$output" == "= __ --a completely different VALuE    " ]]
-
-	sane_run jq -SMr '.annotations["com.cyphar.label_empty"]' "$BUNDLE/config.json"
-	[ "$status" -eq 0 ]
-	[[ "$output" == "" ]]
-
-	sane_run jq -SMr '.annotations["com.cyphar.manifest_empty"]' "$BUNDLE/config.json"
-	[ "$status" -eq 0 ]
-	[[ "$output" == "" ]]
+	# the ports are comma separated
+	[[ "${lines[0]}" == "1234/tcp,8080/tcp" ]]
```

Comment:
If you want to implement this in a way that doesn't hardcode the ordering (not sure if it makes sense here) you could do something like `$(echo "${lines[0]}" | tr ',' '\n' | grep <blah>)` but it's a bit ugly for my taste.